library/src/commonMain/kotlin/com/badoo/reaktive/observable/Various.kt

```diff
@@ -0,0 +1,39 @@
+package com.badoo.reaktive.observable
+
+inline fun <T> observable(crossinline subscribe: (observer: ObservableObserver<T>) -> Unit): Observable<T> =
```

Comment:
There is an extension function subscribe, a method subscribe and a variable function subscribe. It gets pretty confusing, I think it is worth renaming this variable. It currenty looks like the subscribe method recursively calls itself.